.circleci/config.yml

```diff
@@ -31,10 +31,10 @@ commands:
           command: yarn build
       - run:
           name: Run browser tests
-          command: yarn test.karma
+          command: yarn test.karma && USE_NIMMA=true yarn test.karma
```

Comment:
@P0lip Surprisingly the CI builds don't seem to show the same kind of perf improvements when USE_NIMMA=true than we can see running the plain jest tests. Is that because we run less rules under karma context or is the env variable not passed down to code being executed in the browser?

Reply:
Ah, could be the env is not passed down to the browser...
I'm going to take a look and see if that's the case.

Reply:
Yup, you were right